src/EStimLibrary/Extensions/SpatialModel/StringHierarchy/StringHierarchyRegion.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace EStimLibrary.Extensions.SpatialModel.StringHierarchy;
+﻿using System;
```

Comment:
were you getting an error for not having this? It should be found via the global Usings.cs file already

Reply:
No, I was not getting an error. I don't remember adding this, so I will just remove it.